CONTRIBUTING.md

```diff
@@ -54,6 +56,37 @@ Please explain why someone else should merge your proposed change:
 * Does my PR make multiple changes and therefore require multiple commits?
 Ideally, each individual change correlates to an individual commit.
 
+## Capitalization
+
+All headings are sentence case.
+We capitalize the following terms:
+
+* Ansible
+* Candlepin
+* Chef
+* Content View
+* Discovery (service)
+* Hammer
+* Insights (Red Hat product)
+* Kickstart
+* Library (environment)
+* OpenSCAP
+* PostgreSQL
+* Pulp
+* Puppet
+* Python
+* Red Hat Network
+* Redis
+* Salt
+* Salt Minions
+* Tracer (utility)
+
+We do not capitalize the following terms:
+
+* Red Hat subscription manifests
```

Comment:
```suggestion
* Red Hat subscription manifest
```
I think lowercasing the plural will be implied.

Reply:
Fixed.